docs/HOWTO/ie_cpu_extension/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2018-2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# ===============================================================================
+# Generated file for building library with user generated CPU extensions
+#
+# Contains implementation of the basic layer methods
+#
+# Refer to the section "Adding Your Own Kernels to the Inference Engine" in
+# OpenVINO* documentation (either online or offline in
+# <INSTALL_DIR>/deployment_tools/documentation/docs/index.html an then navigate
+# to the corresponding section).
+# ===============================================================================
+
+# [fft_cmake_list:cmake]
+set(CPU_EXTENSIONS_BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR} CACHE INTERNAL "")
```

Comment:
we already have `template_extension` inside `docs` folder. Do we need one more?

Reply:
BTW, if you refer to this extensions in your docs, it seems you have a duplication with https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_Extensibility_DG_Intro.html

CC @ilyachur

Reply:
This file is a part of the sample for the "Custom operation guide" which allows us to run one specific model. And I use different "tags" so there is no conflict/duplication.

Reply:
I agree with Ilya, why cannot we extend `template_extension` library for these cases?
And add new tests to test these kernels.

We have tests for `template_extension` library but looks like we don't validate this code.

Reply:
Done